api/pkg/cmd/stack.go

```diff
@@ -60,6 +60,10 @@ func enrichStacklistMetadata(ctx context.Context, stacklist []string, payload mo
 		integrationSecret.Tfe.Url,
 		integrationSecret.Tfe.Org,
 	).WithTFEToken(setup.GetConfiguration().TFE.Token)
+	ctx, done := context.WithCancel(ctx)
+	defer done()
+	workspace_repo.StartTFCWorkerPool(ctx)
```

Comment:
Perhaps, we should start it globally in one place?

Reply:
I'm about to refactor this, I just want to fix it while I am working on the refactor. The tough part about starting it globally is I need a context object so I can kill the pool. Also the concurrency model is a little different in the API than the CLI, so this is a little bit of a cover while I rework how the concurrency works.